src/controllers/cart.controller.js

```diff
@@ -99,4 +99,26 @@ const clearCart = async (req, res) => {
   }
 };
 
-export { addToCart, viewCart, clearCart };
+const removeFromCart = async (req, res) => {
+  try {
+    const cart = req.cart;
+    const { productId } = req.params;
+    console.log('++++++++++++' + productId);
```

Comment:
- [ ] kindly consider removing this console to make your code clean 
- [ ] How about including the product id in the cart item for it to be easily accessed in the front end ?? when viewing the cart 
like this 
```
     id: product.id
      name: product.name,
      price: product.price,
      quantity: products[index].quantity,
      images: product.images,
      sellerId: product.sellerId,
```

Reply:
for second suggestion I did but I don't think you need that while you have all products fetched in redux store